### ISSUE TITLE
File structure issues, prioritizing

### ISSUE THREAD

**Issue by AvivaShimelman** [lingdb/Sound-Comparisons#326]
Right now, we have a file "3b - GA - To transcribe - Marked Up All 215"

A couple of issues.  Three of those files are not "priority" -- Avava Tembibi, Nahavaq Benour, and Toute Toute. The first two are additional "dialects" of "languages" that are already up on the site; the third is a language that is a priority but not the file we want to put up as a first representative of the language.

Marking them up is certainly not a waste of time -- we will need them eventually -- but we need to subdivide that file into "priority"/"not priority."

**Comment by LauraWae** [comment]
Hi,
I cleaned those up because I was looking for recordings for the website. 
For Tembibi and Banour there were missing soundfiles on the website and I could not find them in our “small” files from last year. The same still goes for wl_0004_Nombotkote_02 (Blacksands) und wl_0036_Neverver_01 (Lingarakh Mindu). Which files correspond then to those languages?
To sum up:
Tembibi, Banour, Blacksands and Lingarakh Mindu still need sound on the website (they do have transcriptions). Which Original sound file name does correspond to them? @AvivaShimelman

**Comment by LauraWae** [comment]
As we have not received it yet, could you please send the Bislama file for cleaning up, too? I think it would be nice to have it on the website alreday.

**Comment by AvivaShimelman** [comment]
Banour is a dialect of Nahavaq (Nahavaq_03 in my system). We already have a
Nahavaq file clean and marked up -- _02 (Caroline Bay dialect).
Tembibi is a dialect of Avava (Avava_04 in my system). The file I have
marked up for Avava _03 (Khatbol dialect).
Nombotkote Blacksands, again, is a dialect of a language for which we
marked up a different file -- (Melaklak (Melkin)).
Same for Neverver 01 -- we're using Neverver 02.
Part of the problem here is Paul's insistence on using "dialect names"
(meaningless in Malakula if not prefixed with language names) without using
language names.  In general, a rule of thumb is simply to look at the
number on the marked up file and go back to the metadata -- on ownCloud.
Also, just keep in mind that, as general rule, we won't have more than one
diaect of a single language, so check if you're doubling on a language
that's already up. I don't know how those got onto the site. I did identify
those in the metadata as recordings that were available, but the very clear
decision was that they wouldn't be going up. Paul asks for metadata on all
recordings, including those that are not immediately going up. I don't know
why he wouldn't weed the list that he feeds into the site, though.

A.

On Sat, Mar 26, 2016 at 8:41 PM, LauraWae <notifications@github.com> wrote:

> Hi,
> I cleaned those up because I was looking for recordings for the website.
> For Tembibi and Banour there were missing soundfiles on the website and I
> could not find them in our “small” files from last year. The same still
> goes for wl_0004_Nombotkote_02 (Blacksands) und wl_0036_Neverver_01
> (Lingarakh Mindu). Which files correspond then to those languages?
> To sum up:
> Tembibi, Banour, Blacksands and Lingarakh Mindu still need sound on the
> website (they do have transcriptions). Which Original sound file name does
> correspond to them? @AvivaShimelman <https://github.com/AvivaShimelman>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/lingdb/Sound-Comparisons/issues/326#issuecomment-201746094>
>



-- 
Aviva Shimelman, PhD

**Comment by AvivaShimelman** [comment]
The files are marked up and the transcriptions are done. In theory, now the
only thing that has to happen before they can go on the site is for me to
double-check them, which will probably take something in the neighborhood
of 2 and a half hours -- I'll put that on my list for when I get back to
Jena. There's nothing wrong with having multiple dialects up now. It's just
that we don't want to do that at the cost of getting examples of "new"
languages up first. It looks like you won't have any problems getting to
those, though, and again, those are site-ready, as soon as I give you the
transcriptions (done, but waiting for a second-look).  i.e, we don't have
to wait for Giovanni to finish his transcription before getting recordings
onto the site.

A.

On Sat, Mar 26, 2016 at 9:51 PM, Aviva Shimelman <nomdecrayon@gmail.com>
wrote:

> Banour is a dialect of Nahavaq (Nahavaq_03 in my system). We already have
> a Nahavaq file clean and marked up -- _02 (Caroline Bay dialect).
> Tembibi is a dialect of Avava (Avava_04 in my system). The file I have
> marked up for Avava _03 (Khatbol dialect).
> Nombotkote Blacksands, again, is a dialect of a language for which we
> marked up a different file -- (Melaklak (Melkin)).
> Same for Neverver 01 -- we're using Neverver 02.
> Part of the problem here is Paul's insistence on using "dialect names"
> (meaningless in Malakula if not prefixed with language names) without using
> language names.  In general, a rule of thumb is simply to look at the
> number on the marked up file and go back to the metadata -- on ownCloud.
> Also, just keep in mind that, as general rule, we won't have more than one
> diaect of a single language, so check if you're doubling on a language
> that's already up. I don't know how those got onto the site. I did identify
> those in the metadata as recordings that were available, but the very clear
> decision was that they wouldn't be going up. Paul asks for metadata on all
> recordings, including those that are not immediately going up. I don't know
> why he wouldn't weed the list that he feeds into the site, though.
>
> A.
>
> On Sat, Mar 26, 2016 at 8:41 PM, LauraWae <notifications@github.com>
> wrote:
>
>> Hi,
>> I cleaned those up because I was looking for recordings for the website.
>> For Tembibi and Banour there were missing soundfiles on the website and I
>> could not find them in our “small” files from last year. The same still
>> goes for wl_0004_Nombotkote_02 (Blacksands) und wl_0036_Neverver_01
>> (Lingarakh Mindu). Which files correspond then to those languages?
>> To sum up:
>> Tembibi, Banour, Blacksands and Lingarakh Mindu still need sound on the
>> website (they do have transcriptions). Which Original sound file name does
>> correspond to them? @AvivaShimelman <https://github.com/AvivaShimelman>
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/lingdb/Sound-Comparisons/issues/326#issuecomment-201746094>
>>
>
>
>
> --
> Aviva Shimelman, PhD
>



-- 
Aviva Shimelman, PhD

**Comment by AvivaShimelman** [comment]
I've been trying to send files for the past couple of days. I'll keep
hitting that one until it goes through. I haven't pre-edited that one
(because it is irrelevant) but I will mark it as "selected" to indicate
that it is ready for cleaning. Just choose whichever sample is clearest.

On Sat, Mar 26, 2016 at 9:25 PM, LauraWae <notifications@github.com> wrote:

> As we have not received it yet, could you please send the Bislama file for
> cleaning up, too? I think it would be nice to have it on the website
> alreday.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/lingdb/Sound-Comparisons/issues/326#issuecomment-201752671>
>



-- 
Aviva Shimelman, PhD

**Comment by AvivaShimelman** [comment]
To be clear: 2 and a half hours per file.

On Sat, Mar 26, 2016 at 9:59 PM, Aviva Shimelman <nomdecrayon@gmail.com>
wrote:

> The files are marked up and the transcriptions are done. In theory, now
> the only thing that has to happen before they can go on the site is for me
> to double-check them, which will probably take something in the
> neighborhood of 2 and a half hours -- I'll put that on my list for when I
> get back to Jena. There's nothing wrong with having multiple dialects up
> now. It's just that we don't want to do that at the cost of getting
> examples of "new" languages up first. It looks like you won't have any
> problems getting to those, though, and again, those are site-ready, as soon
> as I give you the transcriptions (done, but waiting for a second-look).
>  i.e, we don't have to wait for Giovanni to finish his transcription before
> getting recordings onto the site.
>
> A.
>
> On Sat, Mar 26, 2016 at 9:51 PM, Aviva Shimelman <nomdecrayon@gmail.com>
> wrote:
>
>> Banour is a dialect of Nahavaq (Nahavaq_03 in my system). We already have
>> a Nahavaq file clean and marked up -- _02 (Caroline Bay dialect).
>> Tembibi is a dialect of Avava (Avava_04 in my system). The file I have
>> marked up for Avava _03 (Khatbol dialect).
>> Nombotkote Blacksands, again, is a dialect of a language for which we
>> marked up a different file -- (Melaklak (Melkin)).
>> Same for Neverver 01 -- we're using Neverver 02.
>> Part of the problem here is Paul's insistence on using "dialect names"
>> (meaningless in Malakula if not prefixed with language names) without using
>> language names.  In general, a rule of thumb is simply to look at the
>> number on the marked up file and go back to the metadata -- on ownCloud.
>> Also, just keep in mind that, as general rule, we won't have more than one
>> diaect of a single language, so check if you're doubling on a language
>> that's already up. I don't know how those got onto the site. I did identify
>> those in the metadata as recordings that were available, but the very clear
>> decision was that they wouldn't be going up. Paul asks for metadata on all
>> recordings, including those that are not immediately going up. I don't know
>> why he wouldn't weed the list that he feeds into the site, though.
>>
>> A.
>>
>> On Sat, Mar 26, 2016 at 8:41 PM, LauraWae <notifications@github.com>
>> wrote:
>>
>>> Hi,
>>> I cleaned those up because I was looking for recordings for the website.
>>> For Tembibi and Banour there were missing soundfiles on the website and
>>> I could not find them in our “small” files from last year. The same still
>>> goes for wl_0004_Nombotkote_02 (Blacksands) und wl_0036_Neverver_01
>>> (Lingarakh Mindu). Which files correspond then to those languages?
>>> To sum up:
>>> Tembibi, Banour, Blacksands and Lingarakh Mindu still need sound on the
>>> website (they do have transcriptions). Which Original sound file name does
>>> correspond to them? @AvivaShimelman <https://github.com/AvivaShimelman>
>>>
>>> —
>>> You are receiving this because you were mentioned.
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/lingdb/Sound-Comparisons/issues/326#issuecomment-201746094>
>>>
>>
>>
>>
>> --
>> Aviva Shimelman, PhD
>>
>
>
>
> --
> Aviva Shimelman, PhD
>



-- 
Aviva Shimelman, PhD

**Comment by AvivaShimelman** [comment]
Pretty-ing up SC/Malakula --
I'll spend the next couple of days just pecking at everything on the site.
Anything that's homely that's in my power to change, I'll change. Anything
else, I'll list.
Everybody has been working so hard on it, it deserves to be pretty!

On Sat, Mar 26, 2016 at 10:12 PM, Aviva Shimelman <nomdecrayon@gmail.com>
wrote:

> To be clear: 2 and a half hours per file.
>
> On Sat, Mar 26, 2016 at 9:59 PM, Aviva Shimelman <nomdecrayon@gmail.com>
> wrote:
>
>> The files are marked up and the transcriptions are done. In theory, now
>> the only thing that has to happen before they can go on the site is for me
>> to double-check them, which will probably take something in the
>> neighborhood of 2 and a half hours -- I'll put that on my list for when I
>> get back to Jena. There's nothing wrong with having multiple dialects up
>> now. It's just that we don't want to do that at the cost of getting
>> examples of "new" languages up first. It looks like you won't have any
>> problems getting to those, though, and again, those are site-ready, as soon
>> as I give you the transcriptions (done, but waiting for a second-look).
>>  i.e, we don't have to wait for Giovanni to finish his transcription before
>> getting recordings onto the site.
>>
>> A.
>>
>> On Sat, Mar 26, 2016 at 9:51 PM, Aviva Shimelman <nomdecrayon@gmail.com>
>> wrote:
>>
>>> Banour is a dialect of Nahavaq (Nahavaq_03 in my system). We already
>>> have a Nahavaq file clean and marked up -- _02 (Caroline Bay dialect).
>>> Tembibi is a dialect of Avava (Avava_04 in my system). The file I have
>>> marked up for Avava _03 (Khatbol dialect).
>>> Nombotkote Blacksands, again, is a dialect of a language for which we
>>> marked up a different file -- (Melaklak (Melkin)).
>>> Same for Neverver 01 -- we're using Neverver 02.
>>> Part of the problem here is Paul's insistence on using "dialect names"
>>> (meaningless in Malakula if not prefixed with language names) without using
>>> language names.  In general, a rule of thumb is simply to look at the
>>> number on the marked up file and go back to the metadata -- on ownCloud.
>>> Also, just keep in mind that, as general rule, we won't have more than one
>>> diaect of a single language, so check if you're doubling on a language
>>> that's already up. I don't know how those got onto the site. I did identify
>>> those in the metadata as recordings that were available, but the very clear
>>> decision was that they wouldn't be going up. Paul asks for metadata on all
>>> recordings, including those that are not immediately going up. I don't know
>>> why he wouldn't weed the list that he feeds into the site, though.
>>>
>>> A.
>>>
>>> On Sat, Mar 26, 2016 at 8:41 PM, LauraWae <notifications@github.com>
>>> wrote:
>>>
>>>> Hi,
>>>> I cleaned those up because I was looking for recordings for the
>>>> website.
>>>> For Tembibi and Banour there were missing soundfiles on the website and
>>>> I could not find them in our “small” files from last year. The same still
>>>> goes for wl_0004_Nombotkote_02 (Blacksands) und wl_0036_Neverver_01
>>>> (Lingarakh Mindu). Which files correspond then to those languages?
>>>> To sum up:
>>>> Tembibi, Banour, Blacksands and Lingarakh Mindu still need sound on the
>>>> website (they do have transcriptions). Which Original sound file name does
>>>> correspond to them? @AvivaShimelman <https://github.com/AvivaShimelman>
>>>>
>>>> —
>>>> You are receiving this because you were mentioned.
>>>> Reply to this email directly or view it on GitHub
>>>> <https://github.com/lingdb/Sound-Comparisons/issues/326#issuecomment-201746094>
>>>>
>>>
>>>
>>>
>>> --
>>> Aviva Shimelman, PhD
>>>
>>
>>
>>
>> --
>> Aviva Shimelman, PhD
>>
>
>
>
> --
> Aviva Shimelman, PhD
>



-- 
Aviva Shimelman, PhD

**Comment by AvivaShimelman** [comment]
Another part of the issue is that every sound file has two names -- the
original one that I give it and the one it will eventually have on the
site.  Of course, I can't anticipate what name it will have, so we're kind
of stuck with that. As long as the original file name is pegged to the SC
metadata, this shouldn't be a problem, though, and any information on the
files is immediately retrievable from my metadata.

On my head:
I probably should have gotten to my email earlier in February and, then,
changed plans and stayed in town to deal with everything. I was
legitimately "incomunicado" for almost three weeks. But then I shouldn't
have left you hanging for an additional week and a half. I really figured
you were all busy enough and would just hold off on getting to the Malakula
site. Didn't realize you were under so much pressure to get it up.
Smaller mea-culpas:
I probably shouldn't have given place-holder codes for future recordings.
I probably should have held the Taute file until after I would have
collected the set -- especially, since that does look like a non-repeater.
I probably should have foreseen the multiple-languages issue (too much
confusion necessarily here).
There are a dozen more itms that could go on this list.

On Sat, Mar 26, 2016 at 10:12 PM, Aviva Shimelman <nomdecrayon@gmail.com>
wrote:

> To be clear: 2 and a half hours per file.
>
> On Sat, Mar 26, 2016 at 9:59 PM, Aviva Shimelman <nomdecrayon@gmail.com>
> wrote:
>
>> The files are marked up and the transcriptions are done. In theory, now
>> the only thing that has to happen before they can go on the site is for me
>> to double-check them, which will probably take something in the
>> neighborhood of 2 and a half hours -- I'll put that on my list for when I
>> get back to Jena. There's nothing wrong with having multiple dialects up
>> now. It's just that we don't want to do that at the cost of getting
>> examples of "new" languages up first. It looks like you won't have any
>> problems getting to those, though, and again, those are site-ready, as soon
>> as I give you the transcriptions (done, but waiting for a second-look).
>>  i.e, we don't have to wait for Giovanni to finish his transcription before
>> getting recordings onto the site.
>>
>> A.
>>
>> On Sat, Mar 26, 2016 at 9:51 PM, Aviva Shimelman <nomdecrayon@gmail.com>
>> wrote:
>>
>>> Banour is a dialect of Nahavaq (Nahavaq_03 in my system). We already
>>> have a Nahavaq file clean and marked up -- _02 (Caroline Bay dialect).
>>> Tembibi is a dialect of Avava (Avava_04 in my system). The file I have
>>> marked up for Avava _03 (Khatbol dialect).
>>> Nombotkote Blacksands, again, is a dialect of a language for which we
>>> marked up a different file -- (Melaklak (Melkin)).
>>> Same for Neverver 01 -- we're using Neverver 02.
>>> Part of the problem here is Paul's insistence on using "dialect names"
>>> (meaningless in Malakula if not prefixed with language names) without using
>>> language names.  In general, a rule of thumb is simply to look at the
>>> number on the marked up file and go back to the metadata -- on ownCloud.
>>> Also, just keep in mind that, as general rule, we won't have more than one
>>> diaect of a single language, so check if you're doubling on a language
>>> that's already up. I don't know how those got onto the site. I did identify
>>> those in the metadata as recordings that were available, but the very clear
>>> decision was that they wouldn't be going up. Paul asks for metadata on all
>>> recordings, including those that are not immediately going up. I don't know
>>> why he wouldn't weed the list that he feeds into the site, though.
>>>
>>> A.
>>>
>>> On Sat, Mar 26, 2016 at 8:41 PM, LauraWae <notifications@github.com>
>>> wrote:
>>>
>>>> Hi,
>>>> I cleaned those up because I was looking for recordings for the
>>>> website.
>>>> For Tembibi and Banour there were missing soundfiles on the website and
>>>> I could not find them in our “small” files from last year. The same still
>>>> goes for wl_0004_Nombotkote_02 (Blacksands) und wl_0036_Neverver_01
>>>> (Lingarakh Mindu). Which files correspond then to those languages?
>>>> To sum up:
>>>> Tembibi, Banour, Blacksands and Lingarakh Mindu still need sound on the
>>>> website (they do have transcriptions). Which Original sound file name does
>>>> correspond to them? @AvivaShimelman <https://github.com/AvivaShimelman>
>>>>
>>>> —
>>>> You are receiving this because you were mentioned.
>>>> Reply to this email directly or view it on GitHub
>>>> <https://github.com/lingdb/Sound-Comparisons/issues/326#issuecomment-201746094>
>>>>
>>>
>>>
>>>
>>> --
>>> Aviva Shimelman, PhD
>>>
>>
>>
>>
>> --
>> Aviva Shimelman, PhD
>>
>
>
>
> --
> Aviva Shimelman, PhD
>



-- 
Aviva Shimelman, PhD

**Comment by PaulHeggarty** [comment]
Regarding what Aviva writes above:

_Part of the problem here is Paul's insistence on using "dialect names" (meaningless in Malakula if not prefixed with language names) without using language names.
Also, just keep in mind that, as general rule, we won't have more than one diaect of a single language, so check if you're doubling on a language that's already up. I don't know how those got onto the site. I did identify those in the metadata as recordings that were available, but the very clear decision was that they wouldn't be going up. 

Paul asks for metadata on all recordings, including those that are not immediately going up. I don't know why he wouldn't weed the list that he feeds into the site, though._

I think here is the root of so many problems we have had.  Clearly, Aviva is working on assumptions that she things I share, but I certainly do not.  The idea to have only 1 representative up per 'language' was a very, very early thing, and we are way past that now.  The way I see the website is on the contrary that we absolutely *will* and *should*, already, be including multiple dialects of a single language.  We are including as much coverage as we can.  This explains many of the things that Aviva doesn't get about what we are doing, I think.  

Aviva, please try to rethink all that we are doing in these terms:  we absolutely *do* want multiple dialects of the same 'language' to appear on the website.  (Sound Comparisons exists primarily to provide the precision needed to cover dialect-level variation.)  This explains a lot:  such as why we need dialect names, which you consider 

Yes, the priority *in the very early days* was as many distinct languages as possible, but we have the time and resources to do far more than that, and you were sending us far more recordings than that.  I’m very happy that Laura has been editing these files, and will be putting them up as soon as they’re available.  I’m pretty sure that Russell is very keen to demonstrate just how thoroughly you are covering the island, and to demonstrate that by means of Sound Comparisons. 

_Part of the problem here is Paul's insistence on using "dialect names" (meaningless in Malakula if not prefixed with language names) without using language names_

I will insist on both on the website too, from now on.  Until now, without the proper cross-referencing list that we do now have in the languages reference file, it was pretty impossible for me to work out what was what.  I now can, and can rename all varieties on the website as language (dialect).  Is that OK?

Note that iit is just as valid to say that language names too, if used alone, are meaningless without their dialect names.  But again, I can understand that you had a different perspective because you were working on the assumption that we dI’d not want multiple dialects on the website, when we *do*.

**Comment by AvivaShimelman** [comment]
You've made that clear in emails over the past few days. My assumptions
about your assumptions had already changed.
Of course I'm OK with re-naming.
I, too, am very glad that the recordings are being put to good use.

On Wed, Mar 30, 2016 at 1:20 AM, Paul Heggarty <notifications@github.com>
wrote:

> Regarding what Aviva writes above:
>
> _Part of the problem here is Paul's insistence on using "dialect names"
> (meaningless in Malakula if not prefixed with language names) without using
> language names.
> Also, just keep in mind that, as general rule, we won't have more than one
> diaect of a single language, so check if you're doubling on a language
> that's already up. I don't know how those got onto the site. I did identify
> those in the metadata as recordings that were available, but the very clear
> decision was that they wouldn't be going up.
>
> Paul asks for metadata on all recordings, including those that are not
> immediately going up. I don't know why he wouldn't weed the list that he
> feeds into the site, though._
>
> I think here is the root of so many problems we have had. Clearly, Aviva
> is working on assumptions that she things I share, but I certainly do not.
> The idea to have only 1 representative up per 'language' was a very, very
> early thing, and we are way past that now. The way I see the website is on
> the contrary that we absolutely *will* and *should*, already, be
> including multiple dialects of a single language. We are including as much
> coverage as we can. This explains many of the things that Aviva doesn't get
> about what we are doing, I think.
>
> Aviva, please try to rethink all that we are doing in these terms: we
> absolutely *do* want multiple dialects of the same 'language' to appear
> on the website. (Sound Comparisons exists primarily to provide the
> precision needed to cover dialect-level variation.) This explains a lot:
> such as why we need dialect names, which you consider
>
> Yes, the priority *in the very early days* was as many distinct languages
> as possible, but we have the time and resources to do far more than that,
> and you were sending us far more recordings than that. I’m very happy that
> Laura has been editing these files, and will be putting them up as soon as
> they’re available. I’m pretty sure that Russell is very keen to demonstrate
> just how thoroughly you are covering the island, and to demonstrate that by
> means of Sound Comparisons.
>
> *Part of the problem here is Paul's insistence on using "dialect names"
> (meaningless in Malakula if not prefixed with language names) without using
> language names*
>
> I will insist on both on the website too, from now on. Until now, without
> the proper cross-referencing list that we do now have in the languages
> reference file, it was pretty impossible for me to work out what was what.
> I now can, and can rename all varieties on the website as language
> (dialect). Is that OK?
>
> Note that iit is just as valid to say that language names too, if used
> alone, are meaningless without their dialect names. But again, I can
> understand that you had a different perspective because you were working on
> the assumption that we dI’d not want multiple dialects on the website, when
> we *do*.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/lingdb/Sound-Comparisons/issues/326#issuecomment-202915149>
>



-- 
Aviva Shimelman, PhD
